tola/urls.py

```diff
@@ -58,6 +58,9 @@
 urlpatterns += [
     url(r'^jsi18n/$', JavaScriptCatalog.as_view(), name='javascript-catalog'),
 
+    # pnotify prototype
+    url(r'^pnotify-prototype/', TemplateView.as_view(template_name='prototypes/pnotify-prototype.html')),
```

Comment:
Are you leaving this in place on purpose for now, and planning to take it out before we merge into dev?

Reply:
sure. Let me mark it with a TODO